examples/aishell/s1/conf/fbank.conf

```diff
@@ -1,2 +1,3 @@
 --sample-frequency=16000
 --num-mel-bins=80
+--dither=1.0
```

Comment:
1.0 is the default value.

Reply:
Use a default value here is to tell the user this is a configurable value.